packages/cli/test/unit/commands/target/list.test.ts

```diff
@@ -10,7 +10,45 @@ import createLineIterator from 'line-async-iterator';
 import { parseSpacedTableRow } from '../../../helpers/parse-table';
 
 describe('target ls', () => {
-  describe.todo('--next');
```

Comment:
This was incorrect. `target ls` has no `--next`. The collection is not paginated at the server.